Cargo.toml

```diff
@@ -13,3 +13,11 @@ members = [
     "components/merkledb",
 ]
 exclude = [ "exonum/fuzz" ]
+
+[patch.crates-io]
```

Comment:
Looks like using path dependencies in crates leads to some version conflicts. I feel like using `patch` section like this is much more appropriate when we need to use local versions instead of published ones. However, we'll be able to revert this change when `exonum-parameters` will be published (and will depend on a proper version of published Core with dynamic services).

Reply:
I believe `patch` section is ignored if one tries to `install` (or simply use?) a crate from the git source, while specifying dependencies with `path` works fine in this case. If this is true, using `[patch]` should be a temporary solution at best.